README.md

```diff
@@ -4,8 +4,10 @@ A service providing fast event searching, filtering and aggregation on arbitrary
 
 ## Sentry + Snuba
 
-Add the following line to `~/.sentry/sentry.conf.py`:
+Add/change the following lines to `~/.sentry/sentry.conf.py`:
```

Comment:
"to" should probably be "in" here

Reply:
I can always count on Ted. Reliable Ted as he's called.

Reply:
Actually, I'm not sure what is more grammatically correct here on second thought — it'd be "add to" or "change in" but I'm not sure which takes priority when written like this? If it was written as "Add or change the the following lines", I'd think "in" would be the correct one here?

<big>🤔</big>

Reply:
<img width="304" alt="image" src="https://user-images.githubusercontent.com/375744/53598824-87072a80-3b5a-11e9-9f22-4093f9ee5408.png">

Reply:
<img width="208" alt="screenshot 2019-02-28 13 17 24" src="https://user-images.githubusercontent.com/1189446/53599117-4360f080-3b5b-11e9-80d0-ca186f5ffb88.png">
We're not done here. The people need to know what we choose.